subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -1186,3 +1219,28 @@ static uint8_t ll_cis_parameters_validate(uint8_t cis_idx, uint8_t cis_id,
 
 	return BT_HCI_ERR_SUCCESS;
 }
+
+#if defined(CONFIG_BT_CTLR_CONN_ISO_RELIABILITY_POLICY)
+static uint8_t ll_cis_calculate_ft(uint32_t cig_sync_delay, uint32_t iso_interval_us,
+				   uint32_t sdu_interval, uint32_t latency, uint8_t framed)
+{
+	uint32_t tl;
+
+	/* Framed:
+	 *   TL = CIG_Sync_Delay + FT x ISO_Interval + SDU_Interval
+	 *
+	 * Unframed:
+	 *   TL = CIG_Sync_Delay + FT x ISO_Interval - SDU_Interval
+	 */
+	for (uint16_t ft = 1U; ft <= CONFIG_BT_CTLR_CONN_ISO_STREAMS_MAX_FT; ft++) {
+		tl = cig_sync_delay + ft * iso_interval_us +
+							(framed ? sdu_interval : -sdu_interval);
```

Comment:
Can you do `-sdu_interval` without a cast? 

I assume you've tested it to work, but I'm just surprised you can 

But see also my other comment (https://github.com/zephyrproject-rtos/zephyr/pull/60205#discussion_r1265213188) on an alternative way of doing this in a perhaps more readable way :)

Reply:
Agree, it looks more readble. will change.